resources/testdata/glyphs3/WghtVar_GlyphOrder.glyphs

```diff
@@ -17,7 +17,7 @@ name = glyphOrder;
 value = (
 hyphen,
 space,
-exclam
+not-a-valid-name
```

Comment:
IIRC, Glyphs.app used to crash when a glyph name was present only in the glyphOrder custom parameter but not in the actual glyphs source file... Not sure if this is still the case.